foreign_cc/configure.bzl

```diff
@@ -79,11 +79,14 @@ def _create_configure_script(configureParameters):
     prefix = "{} ".format(ctx.expand_location(attrs.tool_prefix, data)) if attrs.tool_prefix else ""
     configure_prefix = "{} ".format(ctx.expand_location(ctx.attr.configure_prefix, data)) if ctx.attr.configure_prefix else ""
 
+    make_path = "nmake.exe" if ctx.attr.nmake else attrs.make_path
+
     for target in ctx.attr.targets:
         # Configure will have generated sources into `$BUILD_TMPDIR` so make sure we `cd` there
-        make_commands.append("{prefix}{make} -C $$BUILD_TMPDIR$$ {target} {args}".format(
+        make_commands.append("{prefix}{make} {flags} {target} {args}".format(
             prefix = prefix,
-            make = attrs.make_path,
+            make = make_path,
+            flags = "-F $$BUILD_TMPDIR$$/makefile" if ctx.attr.nmake else "-C $$BUILD_TMPDIR$$",
```

Comment:
I think `-F` and `-C` are quite different but `-C` should work in both cases (at least I observed this behavior).